packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -496,6 +496,55 @@ abstract class BoxScrollView extends ScrollView {
 /// )
 /// ```
 ///
+/// ## Children's lifecycle
```

Comment:
I think "child elements" is more idiomatic (throughout).

Reply:
Mostly done selectively. Tweaked in cases where I'm referring to the 'child' as a generic concept or where I'm referring to other trees.